enhancements/console/multi-cluster-console.md

```diff
@@ -0,0 +1,366 @@
+---
+title: multi-cluster-console
+authors:
+  - "@spadgett"
+reviewers:
+  - "@TheRealJon"
+  - "@christianvogt"
+  - "@jamestalton"
+  - "@jhadvig"
+  - "@simonpasquier"
+approvers:
+  - "@jwforres"
+creation-date: 2021-09-22
+last-updated: 2021-09-22
+status: provisional
+see-also:
+  - "/enhancements/console/dynamic-plugins.md"
+  - "/enhancements/console/user-settings.md"
+---
+
+# Multi-cluster Console
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+OpenShift console is currently a single-cluster console. Today, users who work
+in environments with multiple clusters must switch between consoles for each
+cluster. We want to make OpenShift console multi-cluster aware. Users should be
+able to visit one console running on the hub cluster and easily switch between
+spoke clusters in the fleet without leaving that console or logging in again.
+
+Additionally, we want to integrate the OpenShift and ACM consoles so that they
+are a single console under a single URL. This hybrid console will have an "All
+clusters" perspective that contains the ACM multi-cluster views. Selecting a
+cluster from the cluster dropdown will drill down into the traditional
+OpenShift console views for that cluster.
+
+## Motivation
+
+To provide a single web console that lets users interact will all clusters in
+the fleet.
+
+### Goals
+
+* The OpenShift console will have a cluster selector that allows you to view
+  resources for any cluster in the fleet.
+* Users will be able to create and edit resources on spoke clusters from the
+  console running on the hub.
+* The console will support viewing metrics for workloads running on spoke
+  clusters.
+* Console will support single sign-on (SSO) to avoid logins every time a user
+  switches clusters.
+
+### Non-Goals
```

Comment:
Wondering if we also should add that we dont wanna solve any console customizations that is set via console-operator (branding, custom logo, custom product name, ...) since the hub cluster will be only doing API requests to the spoke clusters. right ?

Reply:
We plan to use the branding configuration from the hub cluster.